ports/servoshell/egl/ohos/simpleservo.rs

```diff
@@ -53,15 +55,26 @@ pub fn init(
     );
     debug!("Servo commandline args: {:?}", args);
 
+    let config_dir = PathBuf::from(&options.cache_dir).join("servo");
+    debug!("Configs are located at: {:?}", config_dir);
     let _ = crate::prefs::DEFAULT_CONFIG_DIR
-        .set(resource_dir)
+        .set(config_dir.clone())
         .inspect_err(|e| {
             warn!(
                 "Default Prefs Dir already previously filled. Got error {}",
                 e.display()
             );
         });
 
+    // Try copy `prefs.json` from {this.context.resource_prefsDir}/servo/
+    // to `config_dir` if none exist
+    let source_prefs = resource_dir.join("prefs.json");
+    let target_prefs = config_dir.join("prefs.json");
+    if !target_prefs.exists() && source_prefs.exists() {
+        debug!("Copy {:?} to {:?}", source_prefs, target_prefs);
+        fs::copy(&source_prefs, &target_prefs).unwrap_or_else(|e| panic!("Copy failed! {:?}", e));
```

Comment:
`unwrap_or_else` followed by `panic!` should be equivalent to `.expect` I believe. 
But I think we shouldn't panic here, since servo should be able to still run, even without a config dir / preferences.